packages/repo/src/mst/mst.ts

```diff
@@ -200,31 +209,17 @@ export class MST implements DataStore {
   // -------------------
 
   // Return the necessary blocks to persist the MST to repo storage
-  // If the topmost tree only has one entry and it's a subtree, we can eliminate the topmost tree
```

Comment:
From my last MST bughunt adventures: we were trimming the top of the tree before saving, but i've moved that to the delete operation itself since it was screwing up our diff bookkeeping (& it's actually just more intuitive to do it on the operation anyway)